sys-libs/libcxx/libcxx-9999.ebuild

```diff
@@ -9,13 +9,13 @@ EGIT_REPO_URI="http://llvm.org/git/libcxx.git
 	https://github.com/llvm-mirror/libcxx.git"
 CMAKE_MIN_VERSION=3.4.3
 
-[ "${PV%9999}" != "${PV}" ] && SCM="git-r3" || SCM=""
+[[ ${PV} == 9999 ]] && SCM="git-r3" || SCM=""
```

Comment:
s/pattern matching/filename expansion/, I meant. And in any case, the performance difference in this case is negligible, so readability is more important. if you really insist, we can form a new Working Group to decide on this.